spec/lib/utils/job-utils/stdout-helper.js

```diff
@@ -3644,6 +3644,41 @@ module.exports.racadmJobStatusData = 'Security Alert: Certificate is invalid - s
                                     '----------------------------------------------------------\n' +
                                     '\n';
 
+module.exports.ipAddrLinkOutput = '1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default\n'+
```

Comment:
In general we've started just moving the data out to another dedicated file, then loading it in through this file.